projects/com.oracle.truffle.llvm.frontend/com.oracle.truffle.llvm.frontend

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yes, I did. I think it is the file automatically generated by the mx script. I am not sure if .gitignore needs to be updated to prevent its inclusion.